src/Windows/Avalonia.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -1175,10 +1177,8 @@ public struct MOUSEMOVEPOINT
         public static extern bool GetPointerTouchInfoHistory(uint pointerId, ref int entriesCount, [MarshalAs(UnmanagedType.LPArray), In, Out] POINTER_TOUCH_INFO[] touchInfos);
 
         [DllImport("user32.dll")]
-        public static extern bool EnumDisplayMonitors(IntPtr hdc, IntPtr lprcClip,
-                                                      MonitorEnumDelegate lpfnEnum, IntPtr dwData);
-
-        public delegate bool MonitorEnumDelegate(IntPtr hMonitor, IntPtr hdcMonitor, ref Rect lprcMonitor, IntPtr dwData);
+        public static extern bool EnumDisplayMonitors(nint hdc, nint lprcClip,
+            delegate* unmanaged[Cdecl]<nint, nint, nint, nint, int> lpfnEnum, nint dwData);
```

Comment:
All WinAPI functions and callbacks are `__stdcall`

Reply:
Replaced this method with CsWinRT generation.
It doesn't use function pointers unfortunately, but that's fine for now.